test/integration/test_connection_winrm.yml

```diff
@@ -0,0 +1,37 @@
+- hosts: all
```

Comment:
probably better to target the specific hosts/groups that you're trying to test (eg, ```hosts: winrm-pipelining:winrm-no-pipelining``` or ```hosts: winrm```) in case someone's running with a broader inventory.